tickets/src/main/java/no/unit/nva/publication/ticket/update/UpdateTicketHandler.java

```diff
@@ -213,25 +212,25 @@ private void updateUnpublishedFilesToUnpublishable(SortableIdentifier publicatio
         throws NotFoundException {
         var publication = resourceService.getPublicationByIdentifier(publicationIdentifier);
         var updatedPublication = publication.copy()
-                                     .withAssociatedArtifacts(updateUnpublishedFiles(publication))
+                                     .withAssociatedArtifacts(rejectFiles(publication))
                                      .build();
         resourceService.updatePublication(updatedPublication);
     }
 
-    private List<AssociatedArtifact> updateUnpublishedFiles(Publication publication) {
+    private List<AssociatedArtifact> rejectFiles(Publication publication) {
         var associatedArtifacts = publication.getAssociatedArtifacts();
         return associatedArtifacts.stream()
-                   .map(this::rejectIfUnpublished)
+                   .map(UpdateTicketHandler::rejectFile)
                    .toList();
     }
 
-    private AssociatedArtifact rejectIfUnpublished(AssociatedArtifact associatedArtifact) {
-        return switch (associatedArtifact) {
-            case UnpublishedFile unpublishedFile -> unpublishedFile.toUnpublishableFile();
-            case PendingInternalFile pendingInternalFile -> pendingInternalFile.toRejectedFile();
-            case PendingOpenFile pendingOpenFile -> pendingOpenFile.toRejectedFile();
-            case null, default -> associatedArtifact;
-        };
+    //TODO: Remove unpublishable file and logic related to it after we have migrated files
+    private static AssociatedArtifact rejectFile(AssociatedArtifact associatedArtifact) {
+        if (associatedArtifact instanceof UnpublishedFile unpublishedFile) {
+            return unpublishedFile.toUnpublishableFile();
```

Comment:
Handling rejected UnpublishedFile in own if statement, should be removed after migration.